stories/Colors/Colors.jsx

```diff
@@ -1,34 +1,101 @@
 import React from 'react';
 import propTypes from 'prop-types';
+import * as theme from '../../tailwind.js';
+import { processColor, wcagGrade } from '../../src/utils/colors';
+import { compact, toPairs, omit } from 'lodash';
+const ColorValues = ({ color }) => (
+  <small className="group inline-block float-right text-right">
+    <span className="group-hover:hidden">{color.hex}</span>
+    <span className="hidden group-hover:inline-block">
+      rgb(
+      {color.rgba.slice(0, 3).join(',')})
+    </span>
+  </small>
+);
 
 /**
  * A basic square swatch to display a color
  */
 function Swatch(props) {
```

Comment:
Is it worth pulling some of this out into other components to tidy up a little?

Reply:
eh, I debated that but the color swatch isn't meant to be export for use in other apps so I think it's ok